search-api/config.py

```diff
@@ -10,7 +10,20 @@
 BASE_DIR = os.path.abspath(os.path.dirname(__file__))
 
 # # Example DB Configuration
-SQLALCHEMY_DATABASE_URI = f"sqlite:////{BASE_DIR}/db/db.sqlite3"
+
```

Comment:
**Super Small Comment**

Let's avoid unnecessary empty lines - to be consistent with PEP8